azure-devops/azext_devops/dev/team/_format.py

```diff
@@ -215,7 +215,7 @@ def transform_wiki_page_table_output(result):
 def _transform_wiki_page_row(row):
     table_row = OrderedDict()
     table_row['ETag'] = row['eTag']
-    table_row['Git Path'] = row['page']['gitItemPath']
+    table_row['Page Path'] = '\'{}\''.format(row['page']['path'])
```

Comment:
just curious, why are we doing this in table output specifically?

Reply:
![image](https://user-images.githubusercontent.com/11888714/57514199-84c90580-732d-11e9-891d-0b08fa619f53.png)

Reply:
User needs to use the page path in the command not git item path. So I should have always put this in the table output in the first place. See sample above.